pkg/resources/rbac.authorization.k8s.io/v1/clusterrolebinding/validator_test.go

```diff
@@ -0,0 +1,201 @@
+)
+
+var (
+	gvk = metav1.GroupVersionKind{Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "RoleBinding"}
+	gvr = metav1.GroupVersionResource{Group: "rbac.authorization.k8s.io", Version: "v1", Resource: "rolebindings"}
+)
+
+func TestAdmit(t *testing.T) {
+	t.Parallel()
+
+	defaultRoleBinding := &v1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "default",
+		},
+	}
+	emptyLabelsRoleBinding := &v1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: map[string]string{},
+			Name:   "default",
+		},
+	}
+	roleBindingWithOwnerLabel := &v1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "default",
+			Labels: map[string]string{
+				grbOwnerLabel: "grb-owner",
+			},
+		},
+	}
+	roleBindingWithNewOwnerLabel := &v1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "default",
+			Labels: map[string]string{
+				grbOwnerLabel: "new-owner",
+			},
+		},
+	}
+
+	type args struct {
+		oldRB *v1.ClusterRoleBinding
+		newRB *v1.ClusterRoleBinding
+	}
+	type test struct {
+		name    string
+		args    args
+		allowed bool
+	}
+	tests := []test{
+		{
+			name: "updating with 2 nil labels maps allowed",
+			args: args{
+				oldRB: defaultRoleBinding.DeepCopy(),
+				newRB: defaultRoleBinding.DeepCopy(),
+			},
+			allowed: true,
+		},
+		{
+			name: "updating with 2 empty labels maps allowed",
+			args: args{
+				oldRB: emptyLabelsRoleBinding.DeepCopy(),
+				newRB: emptyLabelsRoleBinding.DeepCopy(),
+			},
+			allowed: true,
+		},
+		{
+			name: "updating labels other than grbOwner allowed",
+			args: args{
+				oldRB: roleBindingWithOwnerLabel.DeepCopy(),
+				newRB: &v1.ClusterRoleBinding{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "default",
+						Labels: map[string]string{
+							"new-label":   "test-value",
+							grbOwnerLabel: "grb-owner",
+						},
+					},
+				},
+			},
+			allowed: true,
+		},
+		{
+			name: "adding grb-owner label allowed",
+			args: args{
+				oldRB: defaultRoleBinding.DeepCopy(),
+				newRB: roleBindingWithOwnerLabel.DeepCopy(),
+			},
+			allowed: true,
+		},
+		{
+			name: "adding grb-owner label to empty labels map allowed",
+			args: args{
+				oldRB: emptyLabelsRoleBinding.DeepCopy(),
+				newRB: roleBindingWithOwnerLabel.DeepCopy(),
+			},
+			allowed: true,
+		},
+		{
+			name: "modifying grb-owner label not allowed",
+			args: args{
+				oldRB: roleBindingWithOwnerLabel.DeepCopy(),
+				newRB: roleBindingWithNewOwnerLabel.DeepCopy(),
+			},
+			allowed: false,
+		},
+		{
+			name: "removing grb-owner label not allowed",
+			args: args{
+				oldRB: roleBindingWithOwnerLabel.DeepCopy(),
+				newRB: defaultRoleBinding.DeepCopy(),
+			},
+			allowed: false,
+		},
+		{
+			name: "replacing labels with empty map not allowed",
+			args: args{
+				oldRB: roleBindingWithOwnerLabel.DeepCopy(),
+				newRB: emptyLabelsRoleBinding.DeepCopy(),
+			},
+			allowed: false,
+		},
+		{
+			name: "error getting old and new ClusterRoleBinding from request",
```

Comment:
I might be missing it, but I'm not sure how this test case differs from the last one. Would you mind taking a look at that?

Reply:
this was a left over. Error is tested in `TestAdmin_errors`. I removed it